projects/python-client/src/datapane/client/api/report/core.py

```diff
@@ -465,3 +522,107 @@ def preview(
             formatting: Sets the basic report styling
         """
         self._save(self._preview_file.name, open=open, formatting=formatting)
+
+    ############################################################################
+    # Local served reports
+    def build(
+        self,
+        path: str,
+        formatting: t.Optional[ReportFormatting] = None,
+    ) -> None:
+        """Build a report which can be served by a local http server
+
+        Args:
+            path: File path to store the document
+            formatting: Sets the basic report styling
+        """
+
+        pl_path: Path = Path(path)
+        name = pl_path.stem[:127]
+        bundle_path = pl_path / SERVED_REPORT_BUNDLE_DIR
+        assets_path = pl_path / SERVED_REPORT_ASSETS_DIR
+
+        bundle_path.mkdir(parents=True, exist_ok=True)
+        assets_path.mkdir(parents=True, exist_ok=True)
+
+        self._served_local_writer.assert_bundle_exists()
+
+        # Copy across symlinked report bundle
+        copy_tree(str(self._served_local_writer.assets / "report"), str(bundle_path))
+
+        # Copy across symlinked Vue module
+        copyfile(str(self._served_local_writer.assets / VUE_ESM_FILE), str(pl_path / VUE_ESM_FILE))
```

Comment:
This results in the Vue file not being in the statics directory.

If the report bundle is moved to `./static/report/`, does that result in Vue being requested from `./statics/vue...js`?

Reply:
All that matters is that the vue module is one level up relative to the report dist directory, so yeah that structure should also be do-able.